playground/backend/cmd/server/controller.go

```
@@ -134,6 +137,32 @@ func (controller *playgroundController) GetRunOutput(ctx context.Context, info *
 	return &pipelineResult, nil
 }
 
+// GetLogs is returning logs of execution for specific pipeline by PipelineUuid
+func (controller *playgroundController) GetLogs(ctx context.Context, info *pb.GetLogsRequest) (*pb.GetLogsResponse, error) {
+	pipelineId, err := uuid.Parse(info.PipelineUuid)
+	if err != nil {
+		logger.Errorf("%s: GetLogs(): pipelineId has incorrect value and couldn't be parsed as uuid value: %s", info.PipelineUuid, err.Error())
+		return nil, errors.InvalidArgumentError("GetLogs", "pipelineId has incorrect value and couldn't be parsed as uuid value: "+info.PipelineUuid)
+	}
+	lastIndex, err := code_processing.GetLastIndex(ctx, controller.cacheService, pipelineId, cache.LogsIndex, "GetLogs")
+	if err != nil {
+		return nil, err
+	}
+	logs, err := code_processing.GetProcessingOutput(ctx, controller.cacheService, pipelineId, cache.Logs, "GetLogs")
+	if err != nil {
+		return nil, err
+	}
+	newLogs := ""
+	if len(logs) > lastIndex {
+		newLogs = logs[lastIndex:]
+		_ = utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.LogsIndex, lastIndex+len(newLogs))
```

seems like you always ignore errors from this method why?